install_dependencies_yum.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# SecureTea install Dependencies.
+# Project:
+#     ╔═╗┌─┐┌─┐┬ ┬┬─┐┌─┐╔╦╗┌─┐┌─┐
+#     ╚═╗├┤ │  │ │├┬┘├┤  ║ ├┤ ├─┤
+#     ╚═╝└─┘└─┘└─┘┴└─└─┘ ╩ └─┘┴ ┴
+#     Version: 2.1
+#     Module: SecureTea
+# Attributes:
+#     distros : APT package manager based
+#     files_definition (TYPE): Dependencies installation script
+
+yum check-update
+yum groupinstall -y "Development Tools" "Development Libraries"
+yum install -y python3-setuptools python3-devel libnetfilter_queue libnfnetlink-devel libnetfilter_queue-devel
+yum install -y epel-release
+yum install -y clamav clamav-update
+yum install -y clamav-server clamav-data clamav-update clamav-filesystem clamav clamav-scanner-systemd clamav-devel clamav-lib clamav-server-systemd
+python3 -m pip install -r requirements.txt
+python3 -m pip install git+https://github.com/kti/python-netfilterqueue
```

Comment:
Can we put this to exact versions ?

Reply:
Sure we can do that. But I am not sure that future archives of apt and yum will have those versions by default/will recognize those version from the archives. As such these applications should work with the latest version.

Still : **Should I proceed to make provide versions?**

Reply:
@rejahrehim some versions depend on the OS so it is better we don't specify it!

For example : build-essential=12.1ubuntu2